### DISCUSSION TITLE
Google Photogrammetry

### DISCUSSION THREAD

**Discussion by Jademalo** [derekhe/msfs2020-google-map#45] [Ideas]
Absolutely love this mod, it makes a huge difference around where I live since the bing maps were so out of date. 

Would it be technically possible, and if so are there any plans to implement Google Photogrammetry instead of Bing Photogrammetry?
I assume there might be issues with the terms of service, but I figured it was worth asking.

My whole city has full Photogrammetry support on Google maps, but absolutely nothing on bing. Being able to use the google data would massively improve this whole area.

Thanks!

**Comment by derekhe** [comment]
Currently it is extremely hard to do.
The reason behind is the Google format is different from Bing format, we need realtime conversion of streaming data.
Another part, which is a blocker right now, is the Bing's photogrammetry format is microsoft private format, to decode and encode them would be extremely hard to do unless microsoft describe the format.

**Comment by derekhe** [comment]
Btw, there are tutorials in youtube you can export google earth's data and import into msfs2020 to fix small areas.

**Comment by Jademalo** [comment]
Damn, that's a shame. I assumed it wouldn't be trivial, but figured it was worth asking anyway.

I've just found one of those tutorials you're talking about though, this definitely should be good enough for the time being!
Thanks for letting me know about that!
